### PR TITLE
feat(jQuery): use latest version, not the one bundled with WordPress

### PR DESCRIPTION
**Pull request by domtra**
I know there has been some discussion about removing jQuery completely. However, we should not be using a library with known vulnerabilities. This is the most basic / standard way of getting it to work.

### CODE REVIEW

**Comment by steffenbew** [comment]
@domtra If this is now a simple asset registration, what do you think about removing the jQuery Feature and registering the script in the DocumentDefault?

**Review by dgrdl — COMMENTED** [review]
@domtra isn't the reason we overwrote the original `jquery` hook that there might be plugins in the frontend loading it in and relying on it? Wouldn't this change lead to issues with such plugins due to the global `window.jQuery` and `window.$` being set (without noConflict)?

**Comment by domtra** [comment]
@steffenbew This was my initial idea. I did see some issues with the load order. but i will check this again.

**Comment by domtra** [comment]
@Qakulukiam i do not think we use wordpress plugins for front end javascript dependencies. correct my if i am wrong. 
we should just make sure that if we code for wp-admin, add `jquery` and not `jquery-flynt` as dependency and that it is a different version.
since wp jquery is loded with noConflict, there is no issue when the admin bar is present in the front end.

File: Features/Components/Component.php [hunk 1]
```diff
@@ -21,7 +21,8 @@ public static function enqueueAssets($componentName, array $dependencies = [])
                 array_push($list, $dependency['name']);
             }
             return $list;
-        }, ['jquery']); // jquery as a default dependency
+        }, ['jquery-flynt']); // jquery-flynt as a default dependency
```
Comment (steffenbew) [inline Features/Components/Component.php]: Can this be in the DocumentDefault as well, so everything is in one place and easy to follow? I might not look and expect this in here.

File: Features/Components/Component.php [hunk 2]
```diff
@@ -21,7 +21,8 @@ public static function enqueueAssets($componentName, array $dependencies = [])
...
+        // NOTE: needs to have Flynt's Jquery feature enabled
```
Comment (steffenbew) [inline Features/Components/Component.php]: This is now obsolete, isn't it?

**Pull request closed without merging**